ios/NativeSigner.xcodeproj/project.pbxproj

```diff
@@ -1746,7 +1724,7 @@
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer: hanwen@parity.io (Y66P73ZL35)";
```

Comment:
Not sure this should stay there

Reply:
good catch, this should be removed here, but use the automatic code signing